scipy/spatial/transform/_rotation.pyx

```diff
@@ -1273,6 +1386,159 @@ cdef class Rotation:
         else:
             return cls(quat, normalize=False, copy=False)
 
+    @cython.embedsignature(True)
+    @classmethod
+    def from_davenport(cls, axes, angles, extrinsic, degrees=False):
+        """Initialize from Davenport angles.
+
+        Rotations in 3-D can be represented by a sequence of 3
+        rotations around a sequence of axes.
+
+        The three rotations can either be in a global frame of reference
+        (extrinsic) or in a body centred frame of reference (intrinsic), which
+        is attached to, and moves with, the object under rotation [1]_.
+
+        For Euler angles, we have the following conditions:
+
+            - ``dot(axis1, axis2) = 0``
+            - ``dot(axis2, axis3) = 0``
+            - ``dot(axis1, axis3) = -1``, ``0`` or ``1``
+
+        For Davenport angles, the third relation is relaxed [2]_:
+            - ``dot(axis1, axis3)`` can be any value between -1 and 1.
+
+        Parameters
+        ----------
+        axes : array_like
+            Specifies sequence of axes for rotations. Must be a list of
+            3 arrays of shape (3, 1).
```

Comment:
This will become `up to 3 axes`, right?

Also I believe it's better to explain that each row is an axis or something similar, no need to mention list of arrays.

Reply:
Yes I copied the parameter definition in `from_euler`!